include/grpcpp/impl/codegen/client_context_impl.h

```diff
@@ -310,11 +310,11 @@ class ClientContext {
   /// client’s identity, role, or whether it is authorized to make a particular
   /// call.
   ///
+  /// It is legal to call this only before initial metadata is sent.
```

Comment:
to call this only **once and only** before initial metadata is sent **(i.e., in a PRE_SEND_INITIAL_METADATA interceptor).**

Reply:
This can be called multiple times right? Since this is the client context object, the client code could be calling it once and the interceptors could also be calling it